test/integration/recruiterController.test.js

```diff
@@ -0,0 +1,53 @@
+const chai = require('chai')
+const { expect } = chai
+const chaiHttp = require('chai-http')
+
```

Comment:
Please remove the word `controller` from the filename

Reply:
Done!